includes/deprecated/deprecated.php

```
@@ -114,3 +115,19 @@ function edac_insert_rule_data( $post, $rule, $ruletype, $rule_obj ) {
+ *
+ * @deprecated 1.23.0
+ *
+ * @param int    $post_ID The ID of the post being saved.
+ * @param object $post    The post object being saved.
+ * @param bool   $update  Whether this is an existing post being updated.
+ * @return int   The post ID.
+ */
+function edac_save_post( $post_ID, $post, $update ) {
+	_deprecated_function( __FUNCTION__, '1.23.0', 'EDAC\Admin\Post_Save::save_post' );
+	$post_save = new Post_Save();
+	return $post_save->delete_issue_data_on_post_trashing( $post_ID, $post, $update );  
```

PHPStan (2.0.3)

Result of method EDAC\Admin\Post_Save::delete_issue_data_on_post_trashing() (void) is used.


(method.void)